_includes/tags.html

```diff
@@ -0,0 +1,6 @@
+{% for tag in page.tags %}
+    {% if tag != 'home' %}
+        {% capture tag_name %}{{ tag }}{% endcapture %}
+        <a href="/tag/#{{ tag_name }}" class="badge badge-pil"><code ><nobr>{{ tag_name }}</nobr></code></a>
+    {% endif %}
+{% endfor %}
```

Comment:
Add` end of line` at the end of this and all the other files which don't have it.